pkgs/by-name/to/torrentstream/package.nix

```diff
@@ -0,0 +1,39 @@
+{ buildDotnetModule
+, lib
+, fetchFromGitHub
+, dotnetCorePackages
+}:
```

Comment:
Make sure to order these correctly (last point): https://nixos.org/manual/nixpkgs/stable/#sec-syntax